source/diagrams/manage-diagrams/create-diagrams.txt

```diff
@@ -10,4 +10,46 @@ Create Diagrams
    :depth: 1
    :class: singlecol
 
-Create diagrams.
\ No newline at end of file
+You can create new diagrams on the Mapping tab of Relational Migrator.
+You can also create multiple diagrams and navigate between them to view
+specific entities of your data model. By default, Relational Migrator
+creates the ``Main Diagram`` diagram with a Relational view and a
```

Comment:
We should use the `:guilabel:` directive when referring directly to a UI element.
```suggestion
creates the :guilabel:`Main Diagram` with a Relational view and a
```

Reply:
Done!